### PR TITLE
[enh] Reboot/shutdown from admin interface

### PR DESCRIPTION
**Pull request by opi**
# Screenshots

## Validation

- [x] **Principle agreement** 1/2 : JimboJoe, ljf
- [x] **Quick review** 1/1 : ljf
- [ ] **Simple test** 0,5/1 : JimboJoe (before ljf changes)
- [x] **Deep review** 1/1: JimboJoe

### CODE REVIEW

**Comment by alexAubin** [comment]
Is this ready for reviews ?

**Comment by opi** [comment]
I guess. Maybe some wording and UX still need improvments

**Comment by JimboJoe** [comment]
I changed wordings and layout slightly:

**Comment by opi** [comment]
thanks @JimboJoe . Still 1 quick review and we're good to go ! Every internetcu.be owner will be happy to be able to reboot without unplugging the power cable !

**Comment by zamentur** [comment]
I have made a test on a vagrant I have not been redirected to the connexion page as expected
capture d ecran 2017-10-03 a 19 05 58

File: src/js/yunohost/controllers/tools.js [hunk 1]
```diff
@@ -157,6 +157,37 @@
...
+    app.get('#/tools/reboot/:action', function (c) {
+        var action = c.params['action'].toLowerCase();
+        if (action == 'reboot' || action == 'shutdown') {
+            c.confirm(
+                y18n.t('tools_' + action),
+                // confirm_reboot_action_reboot or confirm_reboot_action_shutdown
+                y18n.t('confirm_reboot_action_' + action),
+                function(){
+                    c.api('/'+action+'?force', function(data) {
+                        c.redirect('#/logout');
```
Comment (zamentur) [inline src/js/yunohost/controllers/tools.js]: We may just refresh all the page or display a message about the extinction ?

**Pull request merged**